life_server/Functions/Systems/fn_vehicleCreate.sqf

```diff
@@ -1,36 +1,95 @@
 /*
     File: fn_vehicleCreate.sqf
     Author: Bryan "Tonic" Boardwine
-
     Description:
-    Answers the query request to create the vehicle in the database.
+    Creates the vehicle and adds a database entry if vehicle is purchased.
 */
-
 params [
-    ["_uid", "", [""]],
-    ["_side", sideUnknown, [west]],
-    ["_vehicle", objNull, [objNull]],
-    ["_color", -1, [0]]
+    ["_unit", objNull, [objNull]],
+    ["_className","",[""]],
+    ["_spawnPoint","",[""]],
+    ["_color", -1, [0]],
+    ["_purchase",false,[false]]
 ];
 
-//Error checks
-if (_uid isEqualTo "" || {_side isEqualTo sideUnknown} || {isNull _vehicle}) exitWith {};
-if (!alive _vehicle) exitWith {};
+if (isNull _unit || {{_className isEqualTo ""} || {_spawnPoint isEqualTo ""}}) exitWith {};
```

Comment:
will error out. either do
`if (isNull _unit || {_className isEqualTo "" || {_spawnPoint isEqualTo ""}}) exitWith {};`
or
`if (isNull _unit || {_className isEqualTo ""} || {_spawnPoint isEqualTo ""}) exitWith {};`

(second one would be the same tho)